### ISSUE TITLE
Add metadata file for availability for PSEO

### ISSUE THREAD

**Issue by srt1** [labordynamicsinstitute/qwi_schemas#118]
Similar to J2JOD, the PSEO would benefit from an additional metadata table indicating which measures (years post-grad) are in-scope for the current release. This will depend on the (first) year of the graduation cohort, length of the cohort (3 for degree level 05-BA/BS; 5 years o/w) , and the states available in the relevant earnings years. 

Relevant variables (tentative):

```
 inst_level
 institution
 degree_level
 grad_cohort
 cohort_years
 geo_level
 geography
 ind_level
 industry
 status_y1
 status_y5
 status_y10
 ```

**Comment by srt1** [comment]
Status flag values:

* -1: year out of range
  * possibly only for right-censored in PSEO, earnings too far out
* 5: does not meet Census standards 
  * missing significant states
  * any left-censored  values
  * right censored ones not beyond overall range of earnings data
* 1: released 
  * counts should be used as reported, though earnings may still be suppressed

**Comment by jodyhoonstarr** [comment]
From Email:

Please add the rows below to the table at:
https://lehd.ces.census.gov/data/schema/latest/lehd_csv_naming.html#_types_and_products

type | product | Description | Explanation | url
pseoe | pseo | Post Secondary Employment Outcomes - Earnings | Counts of employed graduates by institution, with earnings quartiles | https://lehd.ces.census.gov/data/#pseo
pseof | pseo | Post Secondary Employment Outcomes - Flows | Counts of employed graduates by institution, location of work and industry | https://

**Comment by jodyhoonstarr** [comment]
[variables_identifiers_csv.zip](https://github.com/labordynamicsinstitute/qwi_schemas/files/3252718/variables_identifiers_csv.zip)

And a zip containing the two new variable csv files along with the updated lehd_identifiers_pseo. Let me know if any other files need changed.

**Comment by jodyhoonstarr** [comment]
We caught this in other tickets, right @srt1 ?

**Comment by srt1** [comment]
Not really, this was roughly describing a specific new metadata table, which could be useful to include in the release. We're not going to have it fully fleshed out right now, changing it for future.
